torch/fx/experimental/Partitioner.py

```diff
@@ -0,0 +1,151 @@
+from torch.fx import Graph, GraphModule
+from torch.fx import Node
+from typing import Dict, List, Tuple
+"""a graph module may not fit into one backend.
```

Comment:
nit: device is probably more clear than backend, we generally use backend to refer to a type of device. So there will be several devices of a given backend.